index.js

```diff
@@ -23,3 +23,123 @@ hbgMenu.addEventListener('click', () => {
+    featureImage: './image/Snapshoot-Portfolio-2.svg',
+    technologies: ['HTML', 'CSS', 'Javascript', 'github', 'ruby', 'Bootstraps'],
+    liveVersion: '',
+    scource: '',
+  },
+  {
+    name: 'Facebook 360',
+    categories: ['Facebook', 'Full Stack Dev', '2015'],
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    featureImage: './image/Snapshoot-Portfolio-3.svg',
+    technologies: ['HTML', 'CSS', 'Javascript', 'github', 'ruby', 'Bootstraps'],
+    liveVersion: '',
+    scource: '',
+  },
+  {
+    name: 'Uber Navigation',
+    categories: ['Uber', 'Lead Developer', '2015'],
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    featureImage: './image/Snapshoot-Portfolio-3.svg',
+    technologies: ['HTML', 'CSS', 'Javascript', 'github', 'ruby', 'Bootstraps'],
+    liveVersion: '',
+    scource: '',
+  },
```

Comment:
![inconsistency](https://user-images.githubusercontent.com/93039359/208842195-ca6361db-11f4-4049-a3db-7760b362b713.PNG)

- [ ] It's great that you have used array of objects but i found no sequence in project cards. 😨 For example as you can see in the image above ⬆️ that after Tonic Card there is Uber Card but in your array of objects after Tonic card there is Multi-Post Stories 🤔 Kindly fix this issue 👍